linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_ownership/bash/shared.sh

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+for user in $(awk -F':' '{ if ($3 >= {{{ uid_min }}} && $3 != 65534) print $1 }' /etc/passwd); do
+    home_dir=$(getent passwd $user | cut -d: -f6)
+    # Only update the ownership when necessary.
+    find $home_dir -not -user $user -exec chown -f $user {} \;
+done
```

Comment:
I wonder if this is necessary, wasn't the previous approach enough?
In the end all files' ownership need to converge to the primary group, no?
Doing `find` seems more costly then doing `chwon -Rf`